web/styles/settings.css

```diff
@@ -1894,10 +1898,17 @@ label.preferences-radio-choice-label {
 
     &:disabled {
         cursor: not-allowed;
-        background-color: hsl(0deg 0% 93%);
     }
 }
 
+.settings_textarea:disabled {
+    background-color: var(--background-color-disabled-textarea);
+}
+
+.modal_textarea:disabled {
```

Comment:
Have set this so that this matches other modal inputs and have not changed the background color for disabled inputs in settings UI. Although, we do not disable any textarea element in modals right now.